src/ReinforcementLearningZoo/src/experiments/gridworlds/JuliaRL_BasicDQN_EmptyRoom.jl

```diff
@@ -64,16 +64,16 @@ function RLCore.Experiment(
         total_reward_per_episode,
         time_per_step,
         DoEveryNStep() do t, agent, env
-            with_logger(lg) do
-                @info "training" loss = agent.policy.learner.loss
-            end
-        end,
+        with_logger(lg) do
+            @info "training" loss = agent.policy.learner.loss
+        end
+    end,
         DoEveryNEpisode() do t, agent, env
-            with_logger(lg) do
-                @info "training" reward = total_reward_per_episode.rewards[end] log_step_increment =
+        with_logger(lg) do
+            @info "training" reward = total_reward_per_episode.rewards[end] log_step_increment =
                     0
-            end
-        end,
+        end
+    end,
```

Comment:
I undid this change, I assume it was a mistake. Otherwise you can redo it, but it didn't really make any sense to me.

Reply:
Thanks!

Not sure why my VSCode wrongly formatted them. 😢

Reply:
This seems to be a bug in the VSCode Julia plugin: https://github.com/julia-vscode/julia-vscode/issues/1938